operator-pipeline-images/operatorcert/entrypoints/detect_changed_operators.py

```diff
@@ -229,6 +230,80 @@ def union(self) -> set[AffectedCatalog]:
         return self.added | self.modified | self.deleted
 
 
+class ValidationError(Exception):
+    """
+    Exception raised when the result of the detect_changes function
+    violates any of the constraints
+    """
+
+
+def _validate_result(result: dict[str, list[str]]) -> None:
+    """
+    Validate the result of the detect_changes function.
+    Raises a ValidationError if the result violates
+    any of the constraints.
+
+    Args:
+        result: Dictionary with the detected changes
+    """
+    message = ""
+
+    if len(extra_files := result.get("extra_files", [])) > 0:
+        message += f"The PR affects non-operator files: {sorted(extra_files)} "
```

Comment:
Please update the error message with a nicer formatting. This way the error will contain one big line with no linebreaks. Either add a line break at the end of the message or append each message in a list and format it when raising ValidationError

Reply:
Good point, added line-breaks.